tests/test_evaluate.py

```
@@ -0,0 +1,34 @@
+from src.loans.predict_loans import Evaluator
+import pytest
+import pandas as pd
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)